app/services/validators/validation-error-messages.js

```diff
@@ -11,5 +11,6 @@ module.exports = {
   getFutureDateMessage: function (displayName) { return `${displayName} must be in the future` },
   getIsDateLaterThanMessage: function (displayName, options) { return `${displayName} must be after ${options.secondaryDisplayName}` },
   getIsLessThanLengthMessage: function (displayName, options) { return `${displayName} must be shorter than ${options.length} characters` },
-  getIsValidUsernameMessage: function (displayName) { return `${displayName} is invalid` }
+  getIsValidUsernameMessage: function (displayName) { return `${displayName} is invalid` },
+  getIsBooleanMessage: function (displayName, options) { return `${displayName} must be true or false` }
```

Comment:
Is `options` param used for anything?

Reply:
No, its not required for the boolean check.